internal/repository/pokemonRepository.go

```diff
@@ -0,0 +1,134 @@
+package repository
+
+import (
+	"fmt"
+
+	"github.com/McAdam17/2022Q2GO-Bootcamp/internal/entity"
+	"github.com/McAdam17/2022Q2GO-Bootcamp/internal/utils"
+)
+
+type pokemonRepositoryI interface {
```

Comment:
Mismo comment del uso de las interfaces, no se exportan sino que se crean para usarlas como dependencias 
https://medium.com/@felipedutratine/solid-interface-segregation-principle-in-golang-49d4bbb4d3f7